src/elements/Progressbar/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import style from './style.scss';
+
+const cx = classNames.bind(style);
+
+class Progressbar extends React.Component {
+  constructor (props) {
+    super(props);
+    const { percentage, showPercentageNum, className } = props;
+
+    this.percentage = percentage || 0;
+    this.showPercentageNum = showPercentageNum;
+    this.className = className;
+  }
+
+  getBarClass () {
+    return cx(
+      'progress-bar',
+      this.className);
+  }
```

Comment:
why do we need this method?

Reply:
I am thinking in future, as requirements increasing, the progress bar may be complicated. I want to use think function to handle all class names conversion logic. For now, we do don't need this function. Will remove it.